meilisearch-release-changelog/v1.4.0.md

```diff
@@ -63,14 +63,24 @@ This feature can be used together with the `stopWords` and `synonyms` index sett
 Done by @ManyTheFish in #3946
 
 ## Other improvements
-TBD
+
+Meilisearch better hides your data and no longer shows hidden document's fields in error messages. In case you are trying to sort by using a field that is not sortable but there are some of them that are not displayed either, you will see a message like the following:
+
+```
+Available sortable attributes are: price, stock, <..hidden-attributes>.
+```
+
+Done by @vivek-26 in #3945
 
 # Fixes 🐞
 
-* Fix synonyms display (#3946) @ManyTheFish
+  * Fix synonyms display (#3946) @ManyTheFish
+  * Accept the `null` JSON value as a value of the `_vectors` field (#3998) @Kerollmops
+  * Fix the ranking rule by temporarily disabling an assert in the bucket sort algorithm (#4013) @Kerollmops
```

Comment:
@Kerollmops looks like it's v1.3.2 PRs, I mean, they are in the v1.3.2 Milestones. Where is the error, in this changelogs or the milestones linked to the PRs?

Reply:
Indeed, it's a mistake. Those changes are in v1.3.2. Sorry for that!